.circleci/validate_cdk_version.ts

```diff
@@ -2,12 +2,12 @@ import { exec } from 'child_process';
 
 // @aws-cdk/core indicates CDK v1.
 exec('yarn why @aws-cdk/core', (err, stdout, stderr) => {
-  const cdkV1AbsenceIndicator = "We couldn't find a match";
-  if (stdout.toString().includes(cdkV1AbsenceIndicator) || stderr.toString().includes(cdkV1AbsenceIndicator)) {
-    console.log('Success! CDK V1 not found');
-  } else {
+  const cdkV1PresenceIndicator = '@aws-cdk/core';
```

Comment:
I'm assuming old string changed. Could you share example outputs with and without cdk v1 with yarn 2 ?